src/components/AlphabeticalListSection.jsx

```diff
@@ -12,22 +20,27 @@ export const AlphabeticalListSection = ({ letter, poets, activePoet, pathname, s
   );
 
   return (
-    <li key={letter}>
-      <h2 className="body-text"
+    <motion.li key={letter} variants={menuItemVariants} >
+      <h2
+        className="body-text"
         onClick={() => setIsOpen(!isOpen)}
         tabIndex={0}
         onKeyDown={(e) => {
-          if (e.key === 'Enter' || e.key === ' ') {
+          if (e.key === "Enter" || e.key === " ") {
             setIsOpen(!isOpen);
           }
-        }}>
+        }}
+      >
         <PlusButton isActive={isOpen} />
         <span className="letter">{letter}</span>
       </h2>
       {isOpen && (
```

Comment:
Might be interesting to look into https://motion.dev/docs/react-animate-presence so the space for the lists opens & closes smoothly as well.